storage/plugins/book-club/src/PublicController.php

```
@@ -526,6 +547,117 @@ public function changeBookState(ServerRequestInterface $request, ResponseInterfa
+        $isMember = $membership !== null && $membership['status'] === 'active';
+        $canManage = $this->canManage($club);
+        if (!$isMember && !$canManage) {
+            return $this->notFound($response);
+        }
+
+        $states = $this->repo->workflowStates($club);
+        $books = $this->repo->clubBooks((int) $club['id']);
+        if (!$canManage) {
+            $books = array_values(array_filter(
+                $books,
+                static fn(array $b): bool => $b['state'] !== BookClubPlugin::STATE_PENDING
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $b. Configured minimum length is 3. (undefined)

(ShortVariable)